dp/cloud/python/magma/db_service/models.py

```diff
@@ -295,7 +295,7 @@ class DBChannel(Base):
     """
     __tablename__ = "channels"
     id = Column(Integer, primary_key=True, autoincrement=True)
-    cbsd_id = Column(Integer, ForeignKey("cbsds.id", ondelete="CASCADE"))
+    cbsd_id = Column(Integer, ForeignKey("cbsds.id", ondelete="CASCADE"), index=True)
```

Comment:
I think we should also do that for other tables with `ondelete="CASCADE"`, since we've already experienced problems with those (especially with requests)

Reply:
leaving as is as discussed